kv/txn_coord_sender_test.go

```diff
@@ -35,18 +36,50 @@ import (
 	gogoproto "github.com/gogo/protobuf/proto"
 )
 
+// retryableLocalSender provides a retry option in the event of
+// range splits.
```

Comment:
I'm going to leave off the TODO because it's not going to be worth the effort. Using dist sender creates a circular dependency with the server package, which is why we have some test files in this directory declaring `package kv_test`.